tests/ui/const-generics/projection-as-arg-const.rs

```diff
@@ -1,6 +1,7 @@
-// This is currently not possible to use projections as const generics.
-// More information about this available here:
-// https://github.com/rust-lang/rust/pull/104443#discussion_r1029375633
+// build-pass
```

Comment:
imo this should be a revision, we want to keep asserting that aliases dont work as the type of const params without adt_const_param i think